source/usage-examples/find.txt

```diff
@@ -58,17 +61,17 @@ following:
 
 .. code-block:: javascript
 
-   { title: '10 Minutes', imdb: { rating: 7.9, votes: 743, id: 339976 } }
-   { title: '3x3', imdb: { rating: 6.9, votes: 206, id: 1654725 } }
-   { title: '7:35 in the Morning', imdb: { rating: 7.3, votes: 1555, id: 406501 } }
-   { title: '8', imdb: { rating: 7.8, votes: 883, id: 1592502 } }
-   ...
+  { title: '10 Minutes', imdb: { rating: 7.9, votes: 743, id: 339976 } }
+  { title: '3x3', imdb: { rating: 6.9, votes: 206, id: 1654725 } }
+  { title: '7:35 in the Morning', imdb: { rating: 7.3, votes: 1555, id: 406501 } }
+  { title: '8', imdb: { rating: 7.8, votes: 883, id: 1592502 } }
+  ...
```

Comment:
issue: My eyes may be playing tricks on me, but I think this was formatted correctly before. [I believe these are the lines in the file for this PR](https://github.com/mongodb/docs-node/blob/c973085ce4b297f2e5a53c961aa661d6af94cfa8/source/usage-examples/find.txt#L62-L68), it looks like the edit turns this into 2 spaces where as before it was 3.

I feel the [current pages in the master branch are](https://github.com/mongodb/docs-node/blob/master/source/usage-examples/find.txt) correct, and these edits introduce issues.

This applies to all instances of moving the indents in this PR.

Reply:
Apologies, I went to check and you are right. The pipe showing up made me think it was off, but it's just how it is in the node docs.